src/Components/CoinHistoryAndData/CurrencyChart.jsx

```diff
@@ -1,32 +1,46 @@
+import React from "react";
+import { useState, useEffect, useContext } from "react";
 import {
   AreaChart,
   Area,
   XAxis,
   YAxis,
-  CartesianGrid,
   Tooltip,
 } from "recharts";
-
-import React from "react";
-import { useState, useEffect, useContext } from "react";
+import InputLabel from "@mui/material/InputLabel";
```

Comment:
Destructure and import 10,11,12 in same line

Reply:
done